eng/common/cross/toolchain.cmake

```diff
@@ -90,6 +90,9 @@ elseif(CMAKE_SYSTEM_NAME STREQUAL "FreeBSD")
     set(CMAKE_C_COMPILER_TARGET ${triple})
     set(CMAKE_CXX_COMPILER_TARGET ${triple})
     set(CMAKE_ASM_COMPILER_TARGET ${triple})
+    set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -fuse-ld=lld")
```

Comment:
Why is this required? The base compiler already uses `lld` by default. Is for cases when GCC from ports is being used as the compiler?

Reply:
It seems on a Linux cross-build that it's defaulting to the system linker which wasn't creating completely valid FreeBSD objects. The clang LLD linker creates valid FreeBSD objects.

Reply:
Ah, this stuff also works when cross-compiled, wow. The flag makes sense, then.